src/network/MessagePublisher.js

```diff
@@ -1,5 +1,6 @@
 const exchangeName = 'connOut';
 const exchangeControl = 'control';
+const dataExchange = 'data';
```

Comment:
I'd suggest you rename the commit title to explicit this change is related to the message publisher because we can have data adaptations on the message handler too.

Reply:
Done!